salesforce-composite/2.7/modules/ROOT/pages/edits/salesforce-composite-connector-studio.adoc

```diff
@@ -0,0 +1,135 @@
+. On the *General* tab, specify connection information for the connector.
+. On the *TLS* tab, optionally specify the TLS information for the connector.
+. On the *Advanced* tab, optionally specify reconnection information, including a reconnection strategy.
+. Click *Test Connection* to confirm that Mule can connect with the specified server.
+. Click *OK*.
+
+[[add-connector-operation]]
+== Add a Connector Operation to the Flow
+
+When you add a connector operation to your flow, you immediately define a specific operation for that connector to perform.
+
+To add an operation for Salesforce Composite Connector, follow these steps:
+
+. In the *Mule Palette* view, select *Salesforce Composite* and then select the desired operation.
+. Drag the operation onto the Studio canvas to the right of the input source.
+
+[[configure-global-element]]
+== Configure a Global Element for the Connector
+
+When you configure a connector, it’s best to configure a global element that all instances of that connector in the app can use. Configuring a global element requires you to provide the authentication credentials that the connector requires to access the target Salesforce Composite system. Salesforce Composite Connector supports <mention the types of authorization the connector supports>.
+
+To configure the global element for Salesforce Composite Connector, follow these steps:
+
+. Select the name of the connector in the Studio canvas.
+. In the configuration screen for the operation, click the plus sign (+) next to the *Connector configuration* field to access the global element configuration fields.
+. . On the *General* tab, in *Connection*, select the authentication method to configure:
++
+* *OAuth v2.0*
+* *OAuth JWT*
+* *OAuth SAML*
+* *<<oauth-username-password,OAuth Username and Password>>*
++
+You can reference a configuration file that contains ANT-style property placeholders (recommended), or you can enter your authorization credentials in the global configuration properties. For information about the benefits of using property placeholders and how to configure them, see xref:connectors::introduction/intro-connector-configuration-overview.adoc[Anypoint Connector Configuration].
+. On the *Advanced* tab, optionally specify reconnection information, including a reconnection strategy.
+. Click *Test Connection* to confirm that Mule can connect with the specified server.
+. Click *OK*.
+. On the *Advanced* tab, optionally specify reconnection information, including a reconnection strategy.
+. Click *Test Connection* to confirm that Mule can connect with the specified server.
+. Click *OK*.
+
+[[oauth-username-password]]
+=== OAuth Username and Password
+
+Enter the following information on the *General* tab of the *Global Element Properties* screen to configure Basic authentication:
+
+** Consumer Key +
```

Comment:
Why do these begin with a bullet and open-bold but not end with a close-bold?
Did you intends subbullets? Why?